[Server] Reset Database: Panic on Reset

#### Current Behavior
Database reset crashes Meshery Server


#### Screenshots/Logs
```
INFO[0405] kubernetes contexts successfully retrieved from remote provider
interface conversion: error is sqlite3.Error, not *errors.Error

goroutine 1910 [running]:
runtime/debug.Stack()
	/Users/lee/sdk/go1.17.8/src/runtime/debug/stack.go:24 +0x65
runtime/debug.PrintStack()
	/Users/lee/sdk/go1.17.8/src/runtime/debug/stack.go:16 +0x19
github.com/99designs/gqlgen/graphql.DefaultRecover({0xc003e623c0, 0x12}, {0x679f180, 0xc0072568d0})
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/recovery.go:17 +0x71
github.com/99designs/gqlgen/graphql.(*OperationContext).Recover(0x0, {0x70b1cc8, 0xc001bb12c0}, {0x679f180, 0xc0072568d0})
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/context_operation.go:114 +0x3e
github.com/layer5io/meshery/internal/graphql/generated.(*executionContext)._Query_resyncCluster.func1()
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:8056 +0x6a
panic({0x679f180, 0xc0072568d0})
	/Users/lee/sdk/go1.17.8/src/runtime/panic.go:1038 +0x215
github.com/layer5io/meshkit/errors.GetCode({0x7055380, 0xc0072568a0})
	/Users/lee/go/pkg/mod/github.com/layer5io/meshkit@v0.5.26/errors/errors.go:87 +0x90
github.com/layer5io/meshkit/logger.(*Logger).Error(0xc000140788, {0x7055380, 0xc0072568a0})
	/Users/lee/go/pkg/mod/github.com/layer5io/meshkit@v0.5.26/logger/logger.go:71 +0xbb
github.com/layer5io/meshery/internal/graphql/resolver.(*Resolver).resyncCluster(0xc000a4c070, {0x70b1cc8, 0xc001bb12c0}, {0x7152d98, 0xc000602d00}, 0xc001bb11d0, {0x0, 0x0})
	/Users/lee/code/meshery/internal/graphql/resolver/meshsync.go:158 +0x7d0
github.com/layer5io/meshery/internal/graphql/resolver.(*queryResolver).ResyncCluster(0xc0005cab90, {0x70b1cc8, 0xc001bb12c0}, 0x8, {0x0, 0x0})
	/Users/lee/code/meshery/internal/graphql/resolver/schema.resolvers.go:57 +0xa5
github.com/layer5io/meshery/internal/graphql/generated.(*executionContext)._Query_resyncCluster.func2({0x70b1cc8, 0xc001bb12c0})
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:8062 +0x117
github.com/99designs/gqlgen/graphql/executor.processExtensions.func4({0x70b1cc8, 0xc001bb12c0}, 0xc000180c40)
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/executor/extensions.go:72 +0x26
github.com/layer5io/meshery/internal/graphql/generated.(*executionContext)._Query_resyncCluster(0xc001bb3a70, {0x70b1cc8, 0xc001bb10e0}, {0xc004534900, {0x0, 0xc001b9b9e0, 0x30}})
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:8060 +0x217
github.com/layer5io/meshery/internal/graphql/generated.(*executionContext)._Query.func13({0x70b1cc8, 0xc001bb10e0})
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:13108 +0xae
github.com/99designs/gqlgen/graphql/executor.processExtensions.func3({0x70b1cc8, 0xc001bb10e0}, 0xc001bb3af0)
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/executor/extensions.go:69 +0x26
github.com/layer5io/meshery/internal/graphql/generated.(*executionContext)._Query.func14({0x70b1cc8, 0xc001bb10e0})
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:13116 +0x32
github.com/layer5io/meshery/internal/graphql/generated.(*executionContext)._Query.func15()
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:13120 +0x25
github.com/99designs/gqlgen/graphql.(*FieldSet).Dispatch(0xc00345dae0)
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/fieldset.go:34 +0x96
github.com/layer5io/meshery/internal/graphql/generated.(*executionContext)._Query(0xc001bb3a70, {0x70b1cc8, 0xc001bb1050}, {0xc001bb3690, 0x1, 0x1})
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:13448 +0x279
github.com/layer5io/meshery/internal/graphql/generated.(*executableSchema).Exec.func1({0x70b1cc8, 0xc001bb1020})
	/Users/lee/code/meshery/internal/graphql/generated/generated.go:1504 +0x93
github.com/99designs/gqlgen/graphql/executor.(*Executor).DispatchOperation.func1.1.1({0x70b1cc8, 0xc001bb1020})
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/executor/executor.go:108 +0x2b
github.com/99designs/gqlgen/graphql/executor.processExtensions.func2({0x70b1cc8, 0xc001bb1020}, 0x660d4e0)
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/executor/extensions.go:66 +0x26
github.com/99designs/gqlgen/graphql/executor.(*Executor).DispatchOperation.func1.1({0x70b1cc8, 0xc001bb0f60})
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/executor/executor.go:107 +0x113
github.com/99designs/gqlgen/graphql/handler/transport.POST.Do({}, {0x7094c80, 0xc001a18a80}, 0xc001ba8500, {0x70844e0, 0xc000ca4460})
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/handler/transport/http_post.go:55 +0x328
github.com/99designs/gqlgen/graphql/handler.(*Server).ServeHTTP(0xc000ef8140, {0x7094c80, 0xc001a18a80}, 0xc001ba8500)
	/Users/lee/go/pkg/mod/github.com/99designs/gqlgen@v0.17.10/graphql/handler/server.go:120 +0x302
github.com/layer5io/meshery/handlers.(*Handler).GraphqlMiddleware.func1({0x7094c80, 0xc001a18a80}, 0x660e920, 0x701efb0, 0x6743cc0, {0xc001088c60, 0xb4a4cf8})
	/Users/lee/code/meshery/handlers/middlewares.go:215 +0x38
github.com/layer5io/meshery/handlers.(*Handler).MesheryControllersMiddleware.func1({0x7094c80, 0xc001a18a80}, 0xc001ad9300, 0x701ef90, 0x65a1b20, {0x7152d98, 0xc000602d00})
	/Users/lee/code/meshery/handlers/middlewares.go:108 +0x399
github.com/layer5io/meshery/handlers.(*Handler).KubernetesMiddleware.func1({0x7094c80, 0xc001a18a80}, 0xc001ad8300, 0x701f020, 0x6561280, {0x7152d98, 0xc000602d00})
	/Users/lee/code/meshery/handlers/middlewares.go:174 +0xbec
github.com/layer5io/meshery/handlers.(*Handler).SessionInjectorMiddleware.func1({0x7094c80, 0xc001a18a80}, 0xc001ad8100)
	/Users/lee/code/meshery/handlers/middlewares.go:208 +0x516
net/http.HandlerFunc.ServeHTTP(0xc001ada030, {0x7094c80, 0xc001a18a80}, 0xb4a4b60)
	/Users/lee/sdk/go1.17.8/src/net/http/server.go:2047 +0x2f
github.com/layer5io/meshery/handlers.(*Handler).AuthMiddleware.func1({0x7094c80, 0xc001a18a80}, 0xc001ad8100)
	/Users/lee/code/meshery/handlers/middlewares.go:72 +0x173
net/http.HandlerFunc.ServeHTTP(0x70b1cc8, {0x7094c80, 0xc001a18a80}, 0x701ef80)
	/Users/lee/sdk/go1.17.8/src/net/http/server.go:2047 +0x2f
github.com/layer5io/meshery/handlers.(*Handler).ProviderMiddleware.func1({0x7094c80, 0xc001a18a80}, 0xc001ad8000)
	/Users/lee/code/meshery/handlers/middlewares.go:40 +0x376
net/http.HandlerFunc.ServeHTTP(0xc001a9ff00, {0x7094c80, 0xc001a18a80}, 0xc0015439f8)
	/Users/lee/sdk/go1.17.8/src/net/http/server.go:2047 +0x2f
github.com/gorilla/mux.(*Router).ServeHTTP(0xc0005449c0, {0x7094c80, 0xc001a18a80}, 0xc001a9fe00)
	/Users/lee/go/pkg/mod/github.com/gorilla/mux@v1.8.0/mux.go:210 +0x1cf
net/http.serverHandler.ServeHTTP({0x7082190}, {0x7094c80, 0xc001a18a80}, 0xc001a9fe00)
	/Users/lee/sdk/go1.17.8/src/net/http/server.go:2879 +0x43b
net/http.(*conn).serve(0xc0006c94a0, {0x70b1cc8, 0xc000bdc1e0})
	/Users/lee/sdk/go1.17.8/src/net/http/server.go:1930 +0xb08
created by net/http.(*Server).Serve
	/Users/lee/sdk/go1.17.8/src/net/http/server.go:3034 +0x4e8



^C (pressed CTRL+C)


^CINFO[0565] Doing seeded content cleanup...
panic: interface conversion: error is sqlite3.Error, not *errors.Error

goroutine 1 [running]:
github.com/layer5io/meshkit/errors.GetCode({0x7055380, 0xc00666f350})
	/Users/lee/go/pkg/mod/github.com/layer5io/meshkit@v0.5.26/errors/errors.go:87 +0x90
github.com/layer5io/meshkit/logger.(*Logger).Error(0xc000140788, {0x7055380, 0xc00666f350})
	/Users/lee/go/pkg/mod/github.com/layer5io/meshkit@v0.5.26/logger/logger.go:71 +0xbb
main.main()
	/Users/lee/code/meshery/cmd/main.go:274 +0x1d34
exit status 2
make: *** [server] Error 1
```

#### Environment

- **Host OS:** Mac
- **Meshery Server Version:** stable-v0.6.0-rc.6b


<!-- Optional 
#### To Reproduce
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
-->

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

--- a/internal/graphql/resolver/meshsync.go
+++ b/internal/graphql/resolver/meshsync.go
@@ -118,7 +118,7 @@ func (r *Resolver) getMeshSyncStatus(k8sctx models.K8sContext) model.OperatorCon
 func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider, actions *model.ReSyncActions, k8scontextID string) (model.Status, error) {
 	if actions.ClearDb == "true" {
 		// copies the contents .meshery/config/mesherydb.sql to .meshery/config/.archive/mesherydb.sql
-		// then drops all the DB table and then migrate/create tables, missing foreign keys, constraints, columns and indexes. 
+		// then drops all the DB table and then migrate/create tables, missing foreign keys, constraints, columns and indexes.
 		if actions.HardReset == "true" {
 			mesherydbPath := path.Join(utils.GetHome(), ".meshery/config")
 			err := os.Mkdir(path.Join(mesherydbPath, ".archive"), os.ModePerm)
@@ -127,11 +127,11 @@ func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider, 
 			}
 
 			src := path.Join(mesherydbPath, "mesherydb.sql")
-    	dst := path.Join(mesherydbPath, ".archive/mesherydb.sql")
+			dst := path.Join(mesherydbPath, ".archive/mesherydb.sql")
 
 			fin, err := os.Open(src)
 			if err != nil {
-					return "", err
+				return "", err
 			}
 			defer fin.Close()
 

